org.eclipse.lemminx/src/main/resources/org/eclipse/lemminx/services/snippets/processing-instruction-snippets.json

```diff
@@ -0,0 +1,52 @@
+{
+	"Insert XML Model Schema Association": {
+	  "prefix": [
+		"<?xml-model"
+	  ],
+	  "suffix": "?>",
+	  "body": [
+		"<?xml-model href=\"${1:location}\"?>${0}"
+	  ],
+	  "label": "$description",
+	  "description": "Insert XML Model Processing Instruction for Schema Association"
+	},
+	"Insert XML Model Schema Association with Type": {
+	  "prefix": [
+		"<?xml-model"
+	  ],
+	  "suffix": "?>",
+	  "body": [
+		"<?xml-model href=\"${1:location}\" type=\"${2|application/xml,application/dtd,application/relax-ng-compact-syntax|}\"?>${0}"
+	  ],
+	  "label": "$description",
+	  "description": "Insert XML Model Processing Instruction for Schema Association including Type"
+	},
+	"Insert XML Model Schema Association with Schematypens": {
+	  "prefix": [
+		"<?xml-model"
+	  ],
+	  "suffix": "?>",
+	  "body": [
+		"<?xml-model href=\"${1:location}\" schematypens=\"${2:schematypens}\"?>${0}"
```

Comment:
Schematypeens should be a choice

Reply:
Done.

I also added a snippet for both type and schematypens, that works entirely with options and gets the correlation right (in which case use which).  
That's actually a lot nicer and makes the snippet doe type and for schematypens obsolete.

There is just one problem:
![grafik](https://user-images.githubusercontent.com/33053745/83128753-23acd500-a0dc-11ea-9a89-dc00320ee62a.png)
The text is too long, so you can't see what they actually are... any idea how to get around this?

Reply:
I guess what I could do is to give an attribte `title="DTD Schema"`, `title="XSD Schema"`, etc. And if I put that attribute first, that would make it easily readable, what the options mean.